app/roles/openshift-installer/templates/98-cache-disk.yaml.j2

```diff
@@ -0,0 +1,57 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: 98-cache-disk
+spec:
+  config:
+    ignition:
+      version: 3.1.0
+    storage:
+      disks:
+      - device: /dev/{{ openshift_installer_cache_disk }}
+        wipeTable: True
+        partitions:
+        - sizeMiB: 50000
```

Comment:
Idea: use 50% of the disk size (gathered by ansible facts in `ansible_devices`) rather than 50GB statically?
What do you think?

Reply:
Ansible doesn't have any facts about the host at this time. This file is created before the host is created or powered on. I don't think there is a clean way to do that with ipmi/ilo either.